sdk/apimanagement/Azure.ResourceManager.ApiManagement/api/Azure.ResourceManager.ApiManagement.netstandard2.0.cs

```diff
@@ -2638,6 +2638,14 @@ public partial class ApiManagementServiceGetSsoTokenResult
         internal ApiManagementServiceGetSsoTokenResult() { }
         public System.Uri RedirectUri { get { throw null; } }
     }
+    public partial class ApiManagementServiceIdentity
+    {
+        public ApiManagementServiceIdentity(Azure.ResourceManager.ApiManagement.Models.ApimIdentityType apimIdentityType) { }
+        public Azure.ResourceManager.ApiManagement.Models.ApimIdentityType ApimIdentityType { get { throw null; } set { } }
+        public System.Guid? PrincipalId { get { throw null; } }
+        public System.Guid? TenantId { get { throw null; } }
+        public System.Collections.Generic.IDictionary<string, Azure.ResourceManager.Models.UserAssignedIdentity> UserAssignedIdentities { get { throw null; } }
+    }
```

Comment:
This class looks exactly the same as Azure.ResourceManager.Models.ManagedServiceIdentity. We should always have it to be replaced.
If this is related with some nullable property issue, instead of changing the API, we should seek for other options. For instance, we could change the serialization and deserialization of `ManagedServiceIdentity` to make sure it could handle nullable property properly

Reply:
Fxied